mlflow/server/js/src/experiment-tracking/components/ExperimentView.js

```diff
@@ -1214,6 +1244,16 @@ export class ExperimentView extends Component {
     });
   };
 
+  onShare = () => {
+    navigator.clipboard.writeText(window.location.href);
+    message.info(
+      this.props.intl.formatMessage({
+        defaultMessage: 'Experiment view copied to clipboard',
```

Comment:
@marijncv Can we clarify that this is a URL being copied?

```suggestion
        defaultMessage: 'Experiment view URL copied to clipboard',
```